src/app/ui/client-providers.tsx

```diff
@@ -10,7 +11,9 @@ const ClientProviders = ({
   children: React.ReactNode;
 }>) => {
   return (
-    <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
+    <QueryClientProvider client={queryClient}>
+      <Theme>{children}</Theme>
+    </QueryClientProvider>
```

Comment:
these are global providers (e.g. [contexts](https://react.dev/learn/passing-data-deeply-with-context)) that allow any descendant component to access their functionality, like consuming radix-ui theming or react-query storage and utilities